app/api/search.ts

```diff
@@ -49,6 +49,7 @@ export interface SearchResult extends PaginationResponseV2<Paper[]> {
     resultModified: boolean;
     suggestion: Suggestion | null;
     detectedYear: number | null;
+    detectedPhrases: string[];
```

Comment:
is it nullable?

Reply:
No, it always an empty array when there is no item.